CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## v1.13dev
 
+* swapped PyInquirer with questionary for command line questions in launch.py
```

Comment:
```suggestion
* Swapped PyInquirer with questionary for command line questions in `launch.py`
```
On my phone, but would be good to link to the issue here (I'm trying to get into the habit of this - see examples below). Maybe also link to the GitHub pages for the packages.

Reply:
Okay, will do :)